legacy/src/api/payment-requests/payment-requests.md

```diff
@@ -68,6 +68,8 @@ version (documented on this page) and the "legacy" version (documented at
 | -------------------- | ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | patronCodeId         | String             | The id of a [Patron Code][] the payment request is attached to.                                                                                                          |
 | barcode              | String             | [Scanned Code][] used to create the payment request.                                                                                                                     |
+| barcodeType          | String             | The owner or origin of the barcode when the barcode is a token                                                                                                           |
```

Comment:
I don't think we need to have "when the barcode is token"

Reply:
Yep 👍, as discussed together, we will change it to  The identifier of barcode type